src/utils/pickupLocationsUtils.ts

```diff
@@ -0,0 +1,52 @@
+import { type SierraCodeName } from "../types/myAccountTypes"
+
+const cache = {
+  pickupLocations: null,
+  lastUpdated: null,
+  isStillValid: function () {
+    const now = Date.now()
+    return this.lastUpdated && now - this.lastUpdated < 360000
```

Comment:
How'd you get the `360000` number?

Reply:
ope that should be 3600000! for 1 hour

Reply:
Make it a constant `ONE_HOUR` (or something similar) variable to reduce magic numbers and then it's also self-documenting.